common/utils/util.js

```diff
@@ -161,9 +161,10 @@ const formatDateWith = pattern => isoString => {
 const prettyDate = formatDateWith('do MMMM y')
 const prettyDateAndTime = formatDateWith('eeee do MMMM y H:mm')
 
-const ageFrom = (dateOfBirth, today = new Date()) => {
-  const parsedDate = parseISO(dateOfBirth)
-  return isValid(parsedDate) ? Math.abs(differenceInYears(today, parsedDate)) : null
+const ageFrom = (dateOfBirth, now = new Date()) => {
+  const todaysDate = startOfDay(now)
```

Comment:
I've added the additional startOfDay here as technically we were calculating the age on the literal years elapsed, so in theory they could have increased in age mid-way through the day. It's an edge case, but fixed now at least.